go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/Wondertan/iotwal
 go 1.18
 
 require (
-	github.com/celestiaorg/go-libp2p-messenger v0.2.0
+	github.com/celestiaorg/nmt v0.14.0
```

Comment:
Why do we need nmt here lol

Reply:
The first idea was to make a map of mempools with namespace as key. But that was a bad idea.

Reply:
Yes